md_core/commons_svg/bot.py

```
@@ -7,7 +6,7 @@
 import mwclient
 
 sys.argv.append("workibrahem")
-from newapi.page import MainPage
+from mdwiki_api.wiki_page import MainPage  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)